apps/studio-client/shared/js/gcc/src/TranslationServiceSettings.ts

```diff
@@ -0,0 +1,263 @@
+import { as } from "@jangaroo/runtime";
+import { session, Content, ContentRepository, Struct, ContentType } from "@coremedia/studio-client.cap-rest-client";
+import ContentProperties from "@coremedia/studio-client.cap-rest-client/content/ContentProperties";
+import Logger from "@coremedia/studio-client.client-core-impl/logging/Logger";
+import RemoteBeanUtil from "@coremedia/studio-client.client-core/data/RemoteBeanUtil";
+
+/**
+ * Access to the settings for the translation services.
+ *
+ * Note, that this class only provides access to the global settings, not
+ * site specific settings.
+ */
+export class TranslationServicesSettings {
```

Comment:
So there is no prepared SettingsService in the client that could be used?
This can't be the only usage of settings from settings documents in studio-client. Has every use-case their own access and lookup mechanism for settings documents?

Reply:
There are actually multiple solutions for accessing settings documents. The GlobalLink adapter even adds another flavor, which is, that the settings can either be provided as a single document or read from multiple documents.

There is room for optimization. But this needs to be done in the core.